client/scss/overrides/_utilities.focus.scss

```diff
@@ -2,10 +2,15 @@
 // Set global focus outline styles so they are consistent across the UI,
 // without individual components having to explicitly define focus styles.
 // Using !important because we want to enforce only one style is used across the UI.
-.focus-outline-on *:focus {
+//Keeping :focus as a wall back for Safari till release of version 15.4
+*:focus {
   outline: $focus-outline-width solid $color-focus-outline !important;
 }
 
-.focus-outline-off *:focus {
+*:focus:not(:focus-visible) {
   outline: none !important;
 }
+
+*:focus-visible {
+  outline: $focus-outline-width solid $color-focus-outline !important;
+}
```

Comment:
The missing newline here is likely the Prettier formatting issue you were facing – make sure to run Prettier on all changed files before committing code.